src/tactic/suggest.lean

```diff
@@ -384,13 +392,23 @@ matches the goal, and then discharge any new goals using `solve_by_elim`.
 
 If it succeeds, it prints a trace message `exact ...` which can replace the invocation
 of `library_search`.
+
+Once `library_search` applies a lemma from the library it attempts to close any remaining
+goals using `solve_by_elim`. Thus any inputs accepted by `solve_by_elim` are also accepted
+by `suggest`. This allows the user to add lemmas to the list of hypotheses, either as a
+list of lemmas or as a list of attribute names.
```

Comment:
I rewrote this paragraph to give explicit syntax usage.

Reply:
Ok, I also added a similar paragraph to the `suggest` doc string.

Reply:
Um, I already had!?

Reply:
Oh, whoops.